py3status/modules/weather_owm.py

```diff
@@ -273,13 +266,13 @@
 # API information
 OWM_CURR_ENDPOINT = "https://api.openweathermap.org/data/2.5/weather?"
 OWM_FUTURE_ENDPOINT = "https://api.openweathermap.org/data/2.5/forecast?"
-IP_ENDPOINT = "https://timezoneapi.io/api/ip"
+IP_ENDPOINT = "http://geo.ultrabug.fr"
```

Comment:
no `https`? 😛

Reply:
unfortunately no, I can't have one there without paying for it